src/test/java/uk/nhs/tis/trainee/notifications/migration/InsertScheduledEmailHistoryTest.java

```diff
@@ -173,6 +227,33 @@ void shouldInsertScheduledEmailHistory() throws SchedulerException {
     verify(notificationService).saveScheduleHistory(eq(jobDetail2b.getJobDataMap()), any());
   }
 
+  @Test
+  void shouldNotMigrateIfNotificationTypeNotMatch() throws SchedulerException {
```

Comment:
Is there a test that it does not attempt to save scheduled tasks if they are already in mongo? I can't see one.

Reply:
I have add one in the latest commit. Thanks for the reminder ❤️